frontend/src/pages/ProjectSettings/ServicesTable/ServicesTable.tsx

```diff
@@ -0,0 +1,288 @@
+import { Button } from '@components/Button'
+import LoadingBox from '@components/LoadingBox'
+import {
+	useEditServiceGithubSettingsMutation,
+	useGetServicesLazyQuery,
+} from '@graph/hooks'
+import { namedOperations } from '@graph/operations'
+import {
+	Badge,
+	Box,
+	Combobox,
+	IconSolidCubeTransparent,
+	IconSolidExternalLink,
+	IconSolidGithub,
+	Stack,
+	Table,
+	Text,
+	useComboboxState,
+} from '@highlight-run/ui'
+import { useGitHubIntegration } from '@pages/IntegrationsPage/components/GitHubIntegration/utils'
+import { useParams } from '@util/react-router/useParams'
+import { capitalize } from 'lodash'
+import { debounce } from 'lodash'
+import React, { useEffect, useMemo, useState } from 'react'
+
+import { Service, ServiceStatus } from '@/graph/generated/schemas'
+
+import { GitHubSettingsModal } from '../GitHubSettingsModal/GitHubSettingsModal'
+import * as styles from './ServicesTable.css'
+
+type Pagination = {
+	after?: string
+	before?: string
+}
+
+export const ServicesTable: React.FC = () => {
+	const { project_id } = useParams<{ project_id: string }>()
+	const [loadServices, { error, data, loading }] = useGetServicesLazyQuery()
```

Comment:
Do we need a lazy query here? 
I think you can get by with a normal query. There's some gotchas with lazy queries that I tend to avoid them if possible: https://github.com/highlight/highlight/pull/5984#pullrequestreview-1535274804